vocab_files/attribute_concepts/bird-survey-location.ttl

```diff
@@ -0,0 +1,20 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schema: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/78ae22e5-ccf3-4b2c-b07f-1f1120da8d3f>
+    a skos:Concept ;
+    dcterms:source """O’Neill S, McCallum K, Bignall J, Laws M, and Sparrow B, (unpublished draft) ‘Vertebrate Fauna module’ in ….. (eds)
+Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved
+decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:definition "The location where the bird survey weas conducted. Options for 'within survey plot', 'outside survey plot', 'same habitat', 'outside plot, different habitat'" ;
```

Comment:
Shouldn't this be a categorical variable or represented as a concept collection and define each value separately?

Reply:
yes it is categorical with the value type `tern:IRI` and the categorical values collection is https://linked.data.gov.au/def/nrm/2f0470aa-2c05-4e87-a5ac-1d50205b9927